config/packages.yml

```diff
@@ -61,22 +61,8 @@ drupal/acquia_connector:
   version_dev: 4.x-dev
 
 drupal/acquia_contenthub:
-  core_matrix:
-    10.x:
-      version: ~
-      version_dev: ~
-    '*':
-      version: 3.x
-      version_dev: 3.x-dev
-
-drupal/acquia_lift:
-  core_matrix:
-    10.x:
-      version: ~
-      version_dev: ~
-    '*':
-      version: 4.x
-      version_dev: 4.x-dev
```

Comment:
I forget, @secretsayan: why are we removing `drupal/acquia_lift`? I feel l like you told me, but it's been a bit, but it doesn't say anything about it in the ticket or the PR.

Reply:
@TravisCarden : I will mention all the details in the ticket of why we are removing Acquia Lift as they would refer to some internal links.

Reply:
Okay. Lift is being superseded by Perz. Got it; thanks. 👍